ApiReview/API.Polly.Core/NoDocs/Polly.Simmy/BehaviorChaosPipelineBuilderExtensions.cs

```diff
@@ -0,0 +1,14 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Polly.Simmy.Behavior;
+
+namespace Polly.Simmy;
+
+public static class BehaviorChaosPipelineBuilderExtensions
+{
+    public static TBuilder AddChaosBehavior<TBuilder>(this TBuilder builder, bool enabled, double injectionRate, Func<ValueTask> behavior) where TBuilder : ResiliencePipelineBuilderBase;
```

Comment:
```suggestion
    public static TBuilder AddChaosBehavior<TBuilder>(this TBuilder builder, double injectionRate, Func<ValueTask> behavior) where TBuilder : ResiliencePipelineBuilderBase;
```
Do we really need `enabled` parameter, why would I want to add disabled behavior if there is a no way to enable it later?

Applies to other convenience extensions too.

Reply:
We do, it is very important when it comes to enabling/disabling the chaos dynamically, Imagine you have a UI where you can configure the chaos settings, like a master knob to turn it on/off among other settings, I wrote a couple of post about it back then: 
* https://elvanydev.com/chaos-injection-with-simmy/#the-chaos-ui
* https://elvanydev.com/simmy-with-azure-app-configuration/

Reply:
I fully agree with the above, however consider that is more advanced scenario that is fully supported when using options. 

The convenience extensions are just shortcuts and should be (imho) as simple as possible. That's why I suggested dropping the enabled flag as it doesn't add much value, just an additional parameter that is mostly not necessary for the simple scenarios. They can "disable" the behavior by simply setting `injectionRate` to 0 when using these convenience extensions.

Reply:
Oh I see, my bad 🤦‍♂️ I thought you were talking about removing the `enabled` property entirely but you're talking about only the convenience extensions, sorry about it, so I think that makes sense, I'll address your suggestion.

Reply:
done